test/compliance/client-authentication_test.js

```diff
@@ -37,12 +37,14 @@ const client = db.saveClient({ id: 'a', secret: 'b', grants: ['password'] });
 const scope = 'read write';
 
 function createDefaultRequest () {
+  const dice = Math.random() > 0.5;
+  const currentScope = dice ? scope : scope.split(' ');
```

Comment:
I'd suggest we revert this change too, previous tests only accepted strings, so let's keep that so as not to muddy the waters around what is explicitly supported.

IMO it's best to aim for deterministic tests where possible too, so a different approach may be better in the future if array support is added

Reply:
yeah, I removed that